minio/minio
# Added new bucket replication graphs

## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license] (https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
This PR adds new bucket replication graphs for better and granular monitoring of bucket replication. Also arranged all replication graphs together.

## Motivation and Context


## How to test this PR?
Needs https://github.com/minio/minio/pull/17293

### Steps to verify
1. Create two MinIO tenants and setup nucket replication across tenant buckets
2. Load the dashboard and grafana and use bucket metrics scrape point `/minio/v2/metrics/bucket`.
3. Verify that all graphs are displayed as below

![Screenshot from 2023-08-30 12-05-22](https://github.com/minio/minio/assets/6771252/f11dc83d-27b8-42fc-81fa-8cd9d9626f17)


## Types of changes

## Checklist:
